sdk/timeseriesinsights/Instance_API_Design.md

```diff
@@ -0,0 +1,64 @@
+# Time Series Insights
+
+## Instances
+Time Series Model instances are virtual representations of the time series themselves. Instances have descriptive information associated with them called instance properties, such as a time series ID, type, name, description, hierarchies, and instance fields. At a minimum, instance properties include hierarchy information.
+
+### GET /timeseries/instances
+
+```csharp
+public virtual Pageable<TimeSeriesInstance> GetInstances(CancellationToken cancellationToken = default);
+```
+
+### POST /timeseries/instances/suggest
+
+```csharp
+public virtual Response<InstancesSearchStringSuggestion[]> GetSearchSuggestions(InstancesSuggestRequest instancesSuggestRequest, CancellationToken cancellationToken = default);
+
+// Thoughts on renaming InstancesSuggestRequest to GetSearchSuggestionsOptions
```

Comment:
I like "get" better.
one suggestion to add - GetInstancesSearchSuggestion (kind of long though)

Reply:
Thanks for the suggestion. As discussed in the design meeting, I will reach out to TSI team to check whether the suggest API returns more than just intances. In that case, the word instances in the model name doesn't make sense. I will change it to SearchSuggestions for now and will change later.